### ISSUE TITLE
Getting data in WGS84

### ISSUE THREAD

**Issue by math-sasso** [gantian127/soilgrids#3]
It is not clear for me how that could be done:

I would like to get all the brazilian territory in the resolution of 30 seconds (or 0.008333333 in decimal degrees). Which values should I pass as resx, resy, width and height?

I am trying something like that,  but I dont know which should be vaues for 

```python
soil_grids = SoilGrids()
data = soil_grids.get_coverage_data(service_id='phh2o', coverage_id='phh2o_0-5cm_mean', crs='urn:ogc:def:crs:EPSG::4326',
                                        west=-73.981934, south=-33.750035, east=-34.792929, north=5.271478, output='test.tif',
                                        resx=0.008333333 , resy=-0.008333333 , width=39.189005, height=38.991513,
                                        response_crs=None)

for key, value in soil_grids.metadata.items():
    print('{}: {}'.format(key,value))


data.plot(figsize=(9,5))
plt.title('Mean pH between 0 and 5 cm soil depth in Brazil extreme north')
```

And the image I am getting is completely distorted. 

![brazil_soilgrids](https://user-images.githubusercontent.com/23565626/148649033-1bcb8dd1-b875-4d81-93e2-a3611ca6cc9f.png)

**Comment by gantian127** [comment]
If you provide bounding box as lat, lon values, please specify width and height parameters and there is no need to specify resx and resy. The width will be the column number of the tif image and the height value will be row number of the image . So both parameters should be integers. There is an example in the tutorial Jupyter Notebook (section 3) that you will learn how to do it. If you have additional question, please let me know. Thanks!

**Comment by math-sasso** [comment]
Thanks for the explanation @gantian127, it worked well :)  
I suggest you have it clear in the documentation.

Thanks!

**Comment by gantian127** [comment]
Thanks. I will update the documentation to add more details.

**Comment by gantian127** [comment]
This issue is fixed in v0.1.3 with detailed info for "width" and "height".
